src/util.rs

```diff
@@ -1,5 +1,8 @@
 use crate::leader_election::Round;
```

Comment:
Now that we are creating a utils module for config... (see [comment](https://github.com/haraldng/omnipaxos/pull/17#discussion_r685451326) for #17), We might as well move this to `utils/logger.rs`